components/cam/bld/config_files/horiz_grid.xml

```diff
@@ -42,6 +42,8 @@
 <horiz_grid dyn="se"    hgrid="ne240np4"   ncol="3110402"  csne="240"  csnp="4"  />
 <horiz_grid dyn="se"    hgrid="ne0np4_arm_x8v3_lowcon"   ncol="92558"   csne="0"    csnp="4"  />
 <horiz_grid dyn="se"    hgrid="ne0np4_conus_x4v1_lowcon"   ncol="89147"   csne="0"    csnp="4"  />
+<horiz_grid dyn="se"    hgrid="ne0np4_northamerica_x4v1"   ncol="130088"   csne="0"    csnp="4"  />
```

Comment:
Exact same value of ncol for both the North America and Antarctic grids?! That seems unlikely!